keyboards/tweetydabird/lotus58/config.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Markus Knutsson (@TweetyDaBird)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Bootmagic Lite key configuration (these do not transfer to Vial in info.json)
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+#define BOOTMAGIC_LITE_ROW_RIGHT 5
+#define BOOTMAGIC_LITE_COLUMN_RIGHT 0
+
```

Comment:
```suggestion
```

Reply:
I'm leaving these for now, since there are others in there that doesn't work in the .JSON

Reply:
What do you mean they don't work? They auto generated defines from `info.json` work fine from my testing. Or do you meaning something like Vial repository has not adopted these data-driven options..

Reply:
Since the other lines below Aare things not migrated to data driven and these do indeed not work in Vial as data driven (yet), I feel that leaving them all until ALL are data driven should be OK. At that point I'll happily migrate it all.

Reply:
These items should be migrated to info.json, as bootmagic is a feature which is has been covered by data driven for 2 develop cycles.

Unfortunately the whole "vial compatibility" reasoning is non of this repos concern.